dev/com.ibm.websphere.appserver.features/visibility/public/batchManagement-2.0/io.openliberty.batchManagement-2.0.feature

```diff
@@ -0,0 +1,30 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=io.openliberty.batchManagement-2.0
+singleton=true
+visibility=public
+IBM-App-ForceRestart: uninstall, \
+ install
+IBM-API-Package: jakarta.batch.api; type="spec", \
+ jakarta.batch.api.chunk; type="spec", \
+ jakarta.batch.api.chunk.listener; type="spec", \
+ jakarta.batch.api.listener; type="spec", \
+ jakarta.batch.api.partition; type="spec", \
+ jakarta.batch.operations; type="spec", \
+ jakarta.batch.runtime; type="spec", \
+ jakarta.batch.runtime.context; type="spec"
+IBM-ShortName: batchManagement-2.0
+Subsystem-Name: Batch Management 2.0
+-features=com.ibm.websphere.appserver.servlet-5.0, \
+  io.openliberty.batch-2.0, \
+  com.ibm.websphere.appserver.restHandler-1.0, \
+  com.ibm.websphere.appserver.jdbc-4.2; ibm.tolerates:="4.3, 4.1", \
+  io.openliberty.jsonp-2.0, \
+  com.ibm.websphere.appserver.transaction-2.0
```

Comment:
Please add a feature dependency for eeCompatible-9.0.  This is required in all public jakarta features so that WDT can quickly identify jakarta ee9 features.